translations/bnpchs-final-2018-blusukan.html

```diff
@@ -0,0 +1,95 @@
+<!-- Title: Blusukan -->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Bollyjee presidential election is around the corner. Jelix as the presidential 
+  candidate wants to realize the national commitments by doing <i>blusukan</i> (blusukan 
+  is indonesian word that refer to hands-on interaction with public in order to notice their 
+  necessity).
+</p>
+
+<p style="text-align: justify;">
+  There are a bunch of islands and $N$ province in Bollyjee. The island of two provinces lie on could be the 
+  same or different. Jelix had $M$ information, each information provide two provinces 
+  that lie on same island. If province $A$ and $B$ lie on the same island and province $B$ and $C$ lie 
+  on the same island then province $A$ and $C$ also lie on the same island.
+</p>
+
+<p style="text-align: justify;">
+  Each pair of province in the island is connected by a road. Also for each pair of road 
+  there are pavement that connect them so that Jelix can go from a road to other road with this 
+  pavement. Furthermore Jelix can use aerial transportation between islands. He can take off from any island 
+  and land on any island.
+</p>
+
+<p style="text-align: justify;">
+  The program that Jelix uses for blusukan is as follow:
+  <ul>
+  <li>Jelix can start and end his program in any province on any island.</li>
+  <li>Jelix wants to visit <b>all roads</b> in Bollyjee, where he visits every road <b>exactly once</b>.</li>
+  <li>Pavements and roads are different, a visit in pavement is unnecessary and the order of visit is ignored.</li>
+  <li>Because air transports are expensive, Jelix only fly from a island to other island if he has visited all roads in the island.</li>
+  </ul>
+</p>
+
+<p style="text-align: justify;">
+  Jelix requested you to find how many different combination of his road visit, print the value modulo $10^9 + 7$.
```

Comment:
```suggestion
  Jelix requested you to find how many different orders of the roads he will visit. Output the value modulo $10^9 + 7$.
```

kita ga ada ngomongin "kombinasi" apa pun di sini, adanya "urutan"